src/weather.js

```diff
@@ -0,0 +1,24 @@
+/* eslint-disable import/no-cycle, import/no-cycle, import/prefer-default-export */
+import { insertData, changeAppearance } from './ui';
+
+const fetchWeatherData = (location = 'Cairo', units = 'metric') => {
+  const url = `http://api.openweathermap.org/data/2.5/weather?q=${location}&units=${units}&APPID=c0200200bfb501d04806a813e7fcf708`;
+
+  async function getData() {
```

Comment:
Kindly use the ES6 arrow syntax instead :smile:

Reply:
that's An async function declaration